CIP-0155/README.md

```diff
@@ -0,0 +1,133 @@
+
+DP developers SHOULD submit proposals to the SRV Prefix Registry, so SPOs, who deploy them, can have an authoritative information how to do it.
+
+## Specification
+
+### SRV Prefix Registry
+
+| **service** | **scope**     | **SRV prefix**                       | **Status** |
+|:------------|:--------------| ------------------------------------:|:-----------|
+| _cardano_   |               |                      `_cardano._tcp` | Active     |
+| _mithril_   | _dmq_         |       `_.dmq._mithril._cardano._tcp` | Active     |
+| _mithril_   | _aggregator_  | `_aggregator._mithril._cardano._tcp` | Active     |
+
+### SRV Prefix Semantics
+
+When a **cardano** node implementation reads an SRV record from a ledger, it must add the _cardano_ prefix from the table above before making a DNS lookup, e.g. it should do a DNS query for `_cardano._tcp.<srv-record-from-ledger>`.
+
+This design allows DPs to use SRV records registered in the ledger for different purposes, e.g. a **mithril** node can use them to learn about end-points of its network.
+
+Each prefix SHOULD start with `_cardano._tcp` or `_cardano._udp`, to avoid clashes with services not related to `Cardano`.
+
+#### SRV Registry Rules
+
+* Each DP can have at most one entry in the registry.
+* A CIP process assigns new entries, after a careful consideration and consultation with all the involved parties (see #acceptance-criteria below).
+* Entries cannot be removed, but can be revoked by assigning a `Revoked` status.
+  This can only happen if a DP is no longer supported.
+
+### Example
+
+When registering a cardano pool on `example.com` domain using an `SRV` record, one should use:
+```shell
+cardano-cli latest stake-pool ... --multi-host-pool-relay example.com
+```
+(see [register-stake-pool]); and configure SRV record at `_cardano._tcp.example.com` to point to **Cardano** relays, `_mithril._tcp.example.com` to point to **Mithril** relays (see [srv], currently under development).
+
+A **Cardano** node implementation when retrieving information from a registration certificate from the ledger will receive `example.com` and it must prepend the `_cardano._tcp` prefix to make an SRV query.
+
+It may as well append `.` to indicate it is a top-level query, resulting in querying the `_cardano._tcp.example.com.` domain.
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+The CIP should be accepted by:
+
+* [ ] [**Technical Steering Committee**][tsc]
+
+And when there's no major objection from one of the currently involved parties:
+
+* [ ] [**Amaru Team**][amaru] accepts the proposal
+* [ ] [**Cardano-Node Team**][cardano-node] accepts the proposal
+* [ ] [**Gouroboros Team**][gouroboros] accepts the proposal
+* [ ] [**Hydra Team**][hydra] accepts the proposal
+* [ ] [**Mithril Team**][mithril] accepts the proposal
+
+### Implementation Plan
+
+Each **Cardano** node implementation or other tools which rely on SRV records stored in the ledger should comply with this proposal.
```

Comment:
A little more detail here on "comply" would be helpful. What does it mean for Amaru, for example, to "comply" with this proposal?